utils/readers.ts

```diff
@@ -1,19 +1,37 @@
+import { BlobService } from "azure-storage";
 import * as RTE from "fp-ts/ReaderTaskEither";
 import * as TE from "fp-ts/lib/TaskEither";
 import { pipe } from "fp-ts/lib/function";
+
+import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
+import { getBlobAsTextWithError } from "@pagopa/io-functions-commons/dist/src/utils/azure_storage";
+
 import { AssertionRef } from "../generated/definitions/internal/AssertionRef";
 import {
   LolliPOPKeysModel,
   RetrievedLolliPopPubKeys
 } from "../model/lollipop_keys";
-import { cosmosErrorsToString, DomainError, ErrorKind } from "./errors";
+import { AssertionFileName } from "../generated/definitions/internal/AssertionFileName";
+import {
+  cosmosErrorsToString,
+  toInternalError,
+  toNotFoundError,
+  DomainError,
+  ErrorKind
+} from "./errors";
 
 export type PopDocumentReader = RTE.ReaderTaskEither<
   AssertionRef,
   DomainError,
   RetrievedLolliPopPubKeys
 >;
 
+export type AssertionReader = RTE.ReaderTaskEither<
+  AssertionFileName,
+  DomainError,
+  NonEmptyString
+>;
+
 // IMPLEMENTATIONS
```

Comment:
imho this comment is useless.

Reply:
Done in [80a493d](https://github.com/pagopa/io-functions-lollipop/pull/26/commits/80a493db6bd5b57663e3f88ef3461acf83972ec1)